request.go

```diff
@@ -1926,6 +1926,62 @@ func (r *Request) WithMultipart() *Request {
 	return r
 }
 
+//	req := NewRequestC(config, "GET", "/path")
+//	req.WithClient(&http.Client{
+//	  Transport: &http.Transport{
+//		DisableCompression: true,
+//	  },
+//	})
+
+// WithReporter sets reporter.
```

Comment:
Let's clarify this a bit: `WithReporter sets reporter to be use for this request`.

Reply:
Same for WithAssertionHandler